lxd/migrate.go

```diff
@@ -358,7 +367,8 @@ func (s *migrationSourceWs) Do(migrateOp *operation) error {
 		return fsErr
 	}
 
-	if err := s.recv(&header); err != nil {
+	err = s.recv(&header)
+	if err != nil {
```

Comment:
Can you please move those kind of non-functional changes to a separate commit?
This makes it hard to review and will cause a massive headache for stable-2.0 if I can't easily cherry-pick those.